test/rules/no-consecutive-blank-lines/test.ts.lint

```diff
@@ -14,20 +14,20 @@ class Clazz { // comment
 
 ~nil
         console.log("test");
-~nil                         [consecutive blank lines are disallowed]
+~nil [consecutive blank lines are disallowed]
 
     }
 
 
 ~nil
 }
-~nil  [consecutive blank lines are disallowed]
+~nil [consecutive blank lines are disallowed]
 
 //Begin whitespace
 // The next two lines of "code" contain only tabs or spaces, they are also considered "blank" lines
-~nil                         [consecutive blank lines are disallowed]
         
         
+~ [consecutive blank lines are disallowed]
```

Comment:
Why does the test fail here on line 30?  I would expect it to fail on line 28, at the beginning of the blank line sequences.